package.json

```diff
@@ -109,6 +119,7 @@
     "jsdom": "^18.0.0",
     "lint-staged": "11.2.4",
     "prettier": "2.4.1",
+    "storybook-addon-gatsby": "^0.0.2",
```

Comment:
I found the [GH repo] https://github.com/prismicio-community/storybook-addon-gatsby but I wasn't much smarter then from the readme. By looking at the code I see it's about patching the webpack configuration.

> This modifies the Webpack configuration to simulate Gatsby's internal Webpack configuration. It is derived from Gatsby's Visual Testing with Storybook guide and a collective effort of Gatsby users.

It's a contribution from the [prismic](https://prismic.io/docs) folks which have quite some knowledge on gatsby so I trust this to be good enough for our small case 🤓🙌

Reply:
This addon was recommended in the [official Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/testing/visual-testing-with-storybook/#using-an-addon). I noticed that the addon mentions Gatbsy^3 as a peer dependency which made me skeptical at first, but it works fine so I decided to stick with it.